### DISCUSSION TITLE
how to disable floating window on run()

### DISCUSSION THREAD

**Discussion by roychoo** [nvim-neotest/neotest#285] [Q&A]
hi, how do i disable floating window on ```run()```

**Comment by rcarriga** [comment]
I'm assuming this is the `output` window opening on failure. You can set `open_on_run` to `false` to disable. See `:h neotest.Config.output`

**Comment by roychoo** [comment]
actually is it possible to open a split window immediately (at the bottom), with streaming output from the command?

**Reply by gegnew** [reply]
☝️ seconded, can we get a split window like in `ultest`?

**Reply by gegnew** [reply]
or, rather, i understand how to get one for the output window , but i can't seem to configure one for `run.attach()`

for anyone curious, for the output panel:
```
    output_panel = {
      enabled = true,
      open = "botright split | resize 15"
    },
```
